docs/docs/APIdocs/layers/nn/Simple Layers/Reshape.md

```diff
@@ -0,0 +1,50 @@
+## Reshape ##
+
+**Scala:**
+```scala
+val reshape = Reshape(size, batchMode)
+```
+**Python:**
+```python
+reshape = Reshape(size, batch_mode)
+```
+
+The `forward(input)` reshape the input tensor into `size(0) * size(1) * ...` tensor,
+taking the elements row-wise.
+
+**parameters**
+* **size** - the size after reshape
```

Comment:
Not sure if we need to add the parameters details here as user can always get those from API doc. My concern is if we put those parameter descriptions here, it would be hard to sync with the java doc or python doc, and also the naming convention is diff between scala and python. @shane-huang